test/uk/gov/hmrc/tai/model/domain/formatters/EmploymentHodFormattersSpec.scala

```diff
@@ -161,6 +163,14 @@ class EmploymentHodFormattersSpec extends PlaySpec with EmploymentHodFormatters
       parsedJson mustBe samplePayment
     }
 
+    "read nps json and convert it to payment object with duplicate entry" in {
+      val parsedJson: Payment = getJson("rtiInYearFragmentWithDuplicate").as[Payment](paymentHodReads)
```

Comment:
My only problem with testing the json reader in this way is we now have two places where we have to maintain correct json. In here and in the stubs. Have you considered having an integration test instead of this test that tests the relevant connector and deserializes data from the stub

Reply:
Agreed, this is already tested in another connector so I've just deleted the test